src/main/java/org/opentripplanner/middleware/persistence/Persistence.java

```diff
@@ -74,7 +75,8 @@ public static void initialize () {
         apiUsers = new TypedPersistence(mongoDatabase, ApiUser.class);
         tripRequests = new TypedPersistence(mongoDatabase, TripRequest.class);
         tripSummaries = new TypedPersistence(mongoDatabase, TripSummary.class);
-        // TODO Add other models...
+        monitoredTrip = new TypedPersistence(mongoDatabase, MonitoredTrip.class);
```

Comment:
Please pluralize to match other collections.

Reply:
Updated.